catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -119,7 +119,7 @@ public static void processEntry(Metacard metacard, String operation, EventAdmin
             LOGGER.debug("catalog ID = {}", metacard.getId());
             LOGGER.debug("operation = {}", operation);
 
-            HashMap<String, Object> properties = new HashMap<String, Object>();
+            HashMap<String, Object> properties = new HashMap<String, Object>(3, 1);
```

Comment:
I just noticed they were being allocated with the defaults even though we know what the size will be so I fixed it.